ncf-api-virtualenv/debian/dirs

```diff
@@ -1,2 +1,2 @@
 usr/share/ncf-api-virtualenv
-etc/apache2/conf.d
```

Comment:
Why do you want to remove it ? Since you will create a link inside this directory.